### DISCUSSION TITLE
How to find C-index confidence interval in survival random forest in randomForestSRC package.

### DISCUSSION THREAD

**Discussion by AnnamariaP93** [kogalur/randomForestSRC#214] [Q&A] [answered]
I would like to find c-index confidence interval, I have found a solution with bootstrap, but I ask if there is a direct way or a package to do it. Then I would like to plot C-index in some way.

**Comment by ishwaran** [comment, marked as answer]
You can use the [subsample](https://luminwin.github.io/randomForestSRC/reference/subsample.rfsrc.html) function for this.   This function calculates standard errors and confidence regions for variable importance but also has the option `performance` which if set to `TRUE` calculates standard errors and confidence regions for the error rate.

The following code illustrates its use for the [pbc](https://luminwin.github.io/randomForestSRC/reference/pbc.html) data

```
data(pbc, package = "randomForestSRC")
srv.o <- rfsrc(Surv(days, status) ~ ., pbc)
srv.smp.o <- subsample(srv.o, B = 100, performance = TRUE)
plot(srv.smp.o)
     

```

**Reply by AnnamariaP93** [reply]
Thank you, but I need the C-index confidence interval.

**Comment by ishwaran** [comment]
But that's exactly what it does.  It produces a confidence region for the error rate and the error rate is of course the C-index.  Thus it produces a confidence region for the C-index.

**Reply by AnnamariaP93** [reply]
Ah ok. Thank you!
